docs/robotframework-tutorial.rst

```diff
@@ -0,0 +1,382 @@
+========================
+Robot Framework Tutorial
+========================
+
+This tutorial will step you through writing your first test, then
+enhancing that test with a custom keyword implemented as a page
+object. It is not a comprehensive tutorial on using Robot
+Framework. For Robot Framework documentations see the
+`Robot Framework User Guide <http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html>`_
+
+It is assumed you've worked through the CumulusCI :doc:`tutorial` at least up to the
+point where you've called ``cci project init``.
+
+
+
+Part 1: Folder Structure
+========================
+
+We recommend that all robot tests, keywords, data, and log and report files live under
+a folder named ``robot``, at the root of you repository. If you worked
+through the :doc:`CumulusCI tutorial`, the following folders will
+have been created under ``MyProject/robot/MyProject``:
+
+- ``doc`` - a place to put documentation for your tests
+- ``resources`` - a place to put robot libraries and keyword files that
+  are unique to your project
+- ``results`` - a place for robot to write its log and report files
+- ``tests`` - a place for all of your tests.
+
+
+Part 2: Creating a custom object
+================================
+
+For this tutorial we're going to use a custom object named
+``MyObject`` (e.g. ``MyObject__c``). In addition, we need a custom tab that is associated
+with that object.
+
+If you want to run the tests and keywords in this tutorial verbatim,
+you will need to go to Setup and create the following:
+
+1. A custom object with the name ``MyObject``.
+2. A custom tab associated with this object.
+
+
+Part 3: Creating and running your first robot test
+==================================================
+
+The first thing we want to do is create a test that verifies
+we can get to the listing page of the custom object. This will
+let us know that everything is configure properly.
```

Comment:
"configured"

Reply:
@boakley I really like this. It adds a lot of depth and clarity to the existing documentation, as a Robot newbie. I highlighted just one spot that I thought was a little bit unclear above and flagged a few typos.

Reply:
Fixed. Thanks for catching this.